client/src/main/java/org/threadly/litesockets/client/http/HTTPClient.java

```diff
@@ -587,5 +595,22 @@ public MergedByteBuffers getBody() {
     public String getBodyAsString() {
       return body.duplicate().getAsString(body.remaining());
     }
+    
+    public Element getBodyAsXml() throws SAXException {
```

Comment:
I would vote not maintaining it in the library.  But I don't feel strongly about it either way, so I will let you just merge once you are happy with it.